.gitattributes

```diff
@@ -11,3 +11,7 @@ Makefile text
 *.bat eol=crlf
 *.txt text
 *.sh text
+fake_travis_build.sh eol=lf
+.travis.sh eol=lf
+Makefile eol=lf
```

Comment:
IMO this is pattern matching Makefile in any directory, so both match (`/Makefile` is pattern matching top-level one only)